rubicon/java/jni.py

```diff
@@ -15,72 +23,21 @@
 
 # Standard JNI API
 
-java.GetVersion.restype = jint
-java.GetVersion.argtypes = []
-
-java.DefineClass.restype = jclass
-java.DefineClass.argtypes = [c_char_p, jobject, jbyte_p, jsize]
```

Comment:
I'm wondering if we want to keep these unused JNI methods present in the code, but commented out. We don't want to wear the execution overhead of setting up methods we don't need, but having them present means we're acknowledging that they exist but we're *not* using them. Plus, if we ever find ourselves with a need to access one of them, we don't need to go spelunking looking for the docs.

Alternatively, we could add a link to JNI header docs so if we ever need to restore one, we know where we need to start looking. 

Any thoughts/opinions?

Reply:
I have a mild preference for YAGNI here -- link to the docs in a comment I'd be +1 on, and I'd be -1 on having declarations we're not using (but convinceable).

So I like the "link to the JNI header docs" idea quite a bit!

Reply:
I replaced it with a link to the JNI docs.